### PR TITLE
Create new github issue template for security assessments

### PR DESCRIPTION
**Pull request by PushkarJ**
Creates a new security assessment Github issue template for making this a self-service request for any sub-projects

/cc @kubernetes/sig-security-leads @rficcaglia @savitharaghunathan @reylejano 
/sig security 

X-Ref: https://github.com/kubernetes/community/issues/5921

### CODE REVIEW

**Comment by raesene** [comment]
In general this LGTM, one thing do we want to add a field asking the project for contact details of who SIG-Security should get in touch with to progress the request (in case it's not the same as the submitter)

File: .github/ISSUE_TEMPLATE/security-assessment.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+
+<!-- Want to learn more about SIG Security ? 
+
+We started the partnering with maintainers for self-assessments of sub-projects in 2021, to improve security posture and increase awareness of
+security best practices throughout the Kubernetes community
+
+Read about our charter here: https://github.com/kubernetes/community/blob/master/sig-security/charter.md
+
+Have feedback about the process: https://github.com/kubernetes/community/issues/5921
```
Comment (vinayakankugoyal) [inline .github/ISSUE_TEMPLATE/security-assessment.md:34]: nit: add "?"

**Comment by savitharaghunathan** [comment]
> In general this LGTM, one thing do we want to add a field asking the project for contact details of who SIG-Security should get in touch with to progress the request (in case it's not the same as the submitter)

+1. Maybe add a field in the header called 'contact'?

File: .github/ISSUE_TEMPLATE/security-assessment.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+name: Security Assessment Request
+about: Request a Security Assessment from SIG Security members for Kubernetes sub-projects
+title: 'REQUEST: Security Assessment for <sub-project-name>'
+labels: area/security-assessment, sig/security
+assignees:
```
Comment (savitharaghunathan) [inline .github/ISSUE_TEMPLATE/security-assessment.md:6]: Clarifying question: Would this field have info about SIG Security members who are helping with the assessment or would it have a mix of both (requesting K8s sub-project member + SIG security contact) ?

Comment (PushkarJ) [reply .github/ISSUE_TEMPLATE/security-assessment.md:6]: Good question. I actually copied this field from some existing templates here: https://github.com/kubernetes/community/tree/master/.github/ISSUE_TEMPLATE where they have kept it empty too :-)

My guess based on this doc: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository is that it is meant as a field to define a default assignee. But can be safely left blank too.

File: .github/ISSUE_TEMPLATE/security-assessment.md (hunk 1 above)
Comment (PushkarJ) [reply .github/ISSUE_TEMPLATE/security-assessment.md:34]: Updated

**Comment by PushkarJ** [comment]
> > In general this LGTM, one thing do we want to add a field asking the project for contact details of who SIG-Security should get in touch with to progress the request (in case it's not the same as the submitter)
> 
> +1. Maybe add a field in the header called 'contact'?

Updated with a new header asking for this information

**Comment by savitharaghunathan** [comment]
Lgtm

**Review by savitharaghunathan — COMMENTED** [review]
/lgtm
Thank you for creating the template @PushkarJ 😊

**Comment by reylejano** [comment]
/lgtm

Thank you for creating this issue template PR

**Comment by PushkarJ** [comment]
/assign tabbysable coldwater

(because could not get the bot to use the alias of sig-security-pr-reviews)

**Comment by PushkarJ** [comment]
/assign IanColdwater

(did not use the correct handle earlier)

**Comment by PushkarJ** [comment]
https://github.com/kubernetes/test-infra/pull/23070 is now merged (Adding label `area/security-assessment`), so we can use this label in the template. 

@tabbysable @IanColdwater looking for `/approve`. This PR will allow us to start creating a waiting list for security assessments and make it self-service to request one :)

**Comment by tabbysable** [comment]
/approve

Thanks all!

**Pull request merged**